dtm0/dtm0.h

```diff
@@ -91,6 +91,72 @@
  * @{
  */
 
+#include "fid/fid.h"            /* m0_fid */
+#include "fid/fid_xc.h"         /* m0_fid_xc */
+#include "lib/buf.h"            /* m0_bufs */
+#include "lib/buf_xc.h"         /* m0_bufs_xc */
+#include "lib/types.h"          /* uint64_t */
+#include "xcode/xcode.h"        /* M0_XCA_RECORD */
+
+
+struct m0_dtx0_id {
+	struct m0_fid dti_originator_sdev_fid;
+	uint64_t      dti_timestamp;    /* XXX fix the type */
+} M0_XCA_RECORD M0_XCA_DOMAIN(rpc|be);
+
+
+struct m0_dtx0_participants {
+	uint64_t       dtpa_participants_nr;
+	struct m0_fid *dtpa_participants;
+} M0_XCA_SEQUENCE M0_XCA_DOMAIN(rpc|be);
+
+struct m0_dtx0_descriptor {
+	struct m0_dtx0_id           dtd_id;
+	struct m0_dtx0_participants dtd_participants;
```

Comment:
Do you think we can define this as:
```
struct m0_fid_arr dtd_participants;
```

We can keep the code as it is now.

Reply:
Yes.